p2p/src/config.rs

```diff
@@ -19,6 +19,7 @@ use utils::make_config_setting;
 
 use crate::net::types::PubSubTopic;
 
+// TODO: Use different ports on testnet and regtest
 pub const DEFAULT_BIND_PORT: u16 = 3031;
```

Comment:
The ports should be taken from ChainConfig

Reply:
OK, thanks. I added `p2p_port` field to `ChainConfig`.

Reply:
I really remember it was there before... I wonder where it went. Anyway, thank you.